crates/rpc/rpc-types/Cargo.toml

```diff
@@ -13,19 +13,48 @@ Reth RPC types
 [dependencies]
 # reth
 reth-primitives.workspace = true
+reth-codecs = { path = "../../storage/codecs" }
+revm-primitives = { workspace = true, features = ["serde"] }
+
+
+# ethereum
+alloy-primitives = { workspace = true, features = ["rand", "rlp"] }
+alloy-rlp = { workspace = true, features = ["arrayvec"] }
+alloy-sol-types.workspace = true
+ethers-core = { workspace = true, default-features = false, optional = true }
+
+
+# arbitrary utils
+arbitrary = { workspace = true, features = ["derive"], optional = true }
+proptest = { workspace = true, optional = true }
+proptest-derive = { workspace = true, optional = true }
+strum = { workspace = true, features = ["derive"] }
+
+# crypto
+secp256k1 = { workspace = true, features = ["global-context", "recovery"] }
+
+# for eip-4844
+c-kzg = { workspace = true, features = ["serde"] }
```

Comment:
we don't need any of this

the main motivation behind this issue is that we want a standalone new AccessList type

Reply:
> we don't need any of this
> 
> the main motivation behind this issue is that we want a standalone new AccessList type

if i understand very well, we don't want to import B256; from allow_primitive, i need to create our own B256; and Address etc ? Or i'am misundersting something ?

Reply:
no these are just reexported from alloy-primtives, since we still have a dependency on reth_primitives, we can use them, but they can be replaced with alloy_primitives in a followup pr

Reply:
> no these are just reexported from alloy-primtives, since we still have a dependency on reth_primitives, we can use them, but they can be replaced with alloy_primitives in a followup pr

please i'm really sorry, but could you give me example ? i don't really get what i need to change. i'm really sorry

Reply:
of course, so basically import them like this:

https://github.com/paradigmxyz/reth/blob/d2845afdf7482b6e3ff2ded921bdcaffc815a7be/crates/rpc/rpc-types/src/eth/account.rs#L2-L3